### PR TITLE
dhcp: No need to specify /dev/random

### PR DESCRIPTION
**Pull request by firasuke**
/dev/random is the default

### CODE REVIEW

File: srcpkgs/dhcp/template [hunk 1]
```diff
@@ -35,7 +35,7 @@ pre_configure() {
 		--with-srv6-lease-file=/var/lib/dhcp/dhcpd6.leases
 		--with-cli-lease-file=/var/lib/dhclient/dhclient.leases
 		--with-cli6-lease-file=/var/lib/dhclient/dhclient6.leases
-		--with-randomdev=/dev/random"
+		--with-randomdev"
```
Comment (ericonr) [inline srcpkgs/dhcp/template:38]: I'm not sure what this gets us. Would it be a guarantee so an upstream change to `randomdev` is propagated?

**Review by ericonr — COMMENTED** [review]
Please follow the commit style: `dhcp: ....`

File: srcpkgs/dhcp/template (hunk 1 above)
Comment (firasuke) [reply srcpkgs/dhcp/template:38]: > Please follow the commit style: `dhcp: ....`

Done

> I'm not sure what this gets us. Would it be a guarantee so an upstream change to randomdev is propagated?

A tiny bit cleaner template file, and of course it doesn't guarantee what upstream does or doesn't do, but it's unlikely that the default value will change anytime soon (a better value would be `/dev/urandom` but then again we're not debating that).

**Comment by Duncaen** [comment]
Yea not sure if this change is worth it at all, the back and forth about the commit message etc. it really doesn't matter.

**Comment by firasuke** [comment]
Alright then, closing.

**Pull request closed without merging**